subprojects/detekt.yml

```diff
@@ -0,0 +1,689 @@
+        allowedPatterns: ''
+    ForbiddenImport:
+        active: false
+        imports: []
+        forbiddenPatterns: ''
+    ForbiddenMethodCall:
+        active: false
+        methods: ['kotlin.io.println', 'kotlin.io.print']
+    ForbiddenPublicDataClass:
+        active: false
+        ignorePackages: ['*.internal', '*.internal.*']
+    ForbiddenVoid:
+        active: false
+        ignoreOverridden: false
+        ignoreUsageInGenerics: false
+    FunctionOnlyReturningConstant:
+        active: false
+        ignoreOverridableFunction: true
+        excludedFunctions: 'describeContents'
+        excludeAnnotatedFunction: ['dagger.Provides']
+    LibraryCodeMustSpecifyReturnType:
+        active: false
+    LibraryEntitiesShouldNotBePublic:
+        active: false
+    LoopWithTooManyJumpStatements:
+        active: false
+        maxJumpCount: 1
+    MagicNumber:
+        active: false
+        excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**']
+        ignoreNumbers: ['-1', '0', '1', '2']
+        ignoreHashCodeFunction: true
+        ignorePropertyDeclaration: false
+        ignoreLocalVariableDeclaration: false
+        ignoreConstantDeclaration: true
+        ignoreCompanionObjectPropertyDeclaration: true
+        ignoreAnnotation: false
+        ignoreNamedArgument: true
+        ignoreEnums: false
+        ignoreRanges: false
+    MandatoryBracesIfStatements:
+        active: false
+    MandatoryBracesLoops:
+        active: false
+    MaxLineLength:
+        active: false
+        maxLineLength: 120
+        excludePackageStatements: true
+        excludeImportStatements: true
+        excludeCommentStatements: false
+    MayBeConst:
+        active: false
+    ModifierOrder:
+        active: false
+    NestedClassesVisibility:
+        active: false
+    NewLineAtEndOfFile:
+        active: false
+    NoTabs:
+        active: false
+    OptionalAbstractKeyword:
+        active: false
+    OptionalUnit:
+        active: false
+    OptionalWhenBraces:
+        active: false
+    PreferToOverPairSyntax:
+        active: false
+    ProtectedMemberInFinalClass:
+        active: false
+    RedundantExplicitType:
+        active: false
+    RedundantHigherOrderMapUsage:
+        active: false
+    RedundantVisibilityModifierRule:
+        active: false
+    ReturnCount:
+        active: false
+        max: 2
+        excludedFunctions: 'equals'
+        excludeLabeled: false
+        excludeReturnFromLambda: true
+        excludeGuardClauses: false
+    SafeCast:
+        active: false
+    SerialVersionUIDInSerializableClass:
+        active: false
+    SpacingBetweenPackageAndImports:
+        active: false
+    ThrowsCount:
+        active: false
+        max: 2
+    TrailingWhitespace:
+        active: false
+    UnderscoresInNumericLiterals:
+        active: false
+        acceptableDecimalLength: 5
+    UnnecessaryAbstractClass:
+        active: false
```

Comment:
could we use active = false by default?
I want to see what is enabled ASAP

Reply:
as far as i know detekt doesn't allow this